### DISCUSSION TITLE
How to get YOLO achievement?

### DISCUSSION THREAD

**Discussion by rizkyhaksono** [community/community#20420] [Other Feature Feedback, Questions, & Ideas] [answered]
I'm new, so how do I get the badge?

**Comment by mustafaergul** [comment]
https://github.com/github-community/community/discussions/18542#discussioncomment-2974498

**Reply by rizkyhaksono** [reply]
Thanks a lot!

**Reply by AntonAduisky** [reply]
it's already too late

**Reply by isharax9** [reply]
Here is the answer short and sweet, 

1. Create a new pull request in your repo and open it
2. In the top right corner at Reviewers section, request someone to review your pull request
3. Merge the pull request without finishing the review.

Please mark this as answer if it helps ❤️

**Comment by cal-dev-lab** [comment]
Basically, choose a repo, make a branch, make changes in that branch, create a pull request with that branch, add a reviewer to it, merge the pull request before reviewer reviews it, hope this helps :)

**Reply by rizkyhaksono** [reply]
Thanks a lot!

**Comment by cal-dev-lab** [comment]
Please also see here for all achievements! 👍 
https://github.com/Schweinepriester/github-profile-achievements

**Comment by coreymtaylor** [comment]
https://github.com/Schweinepriester/github-profile-achievements

**Reply by cal-dev-lab** [reply]
Nice

**Comment by thuongtruong1009** [comment]
Hi @rizkyhaksono 

* The simple way to get this badge is that you create new PR on your repository, then you merge to the main branch (not make other actions like review, assign, label...) . And you can wait some minutes for Github to update the badge for you!

**Reply by ghost** [reply]
hello @thuongtruong1009  i want to ask you a question... github was made in 2008 and in your account first commit was 1990 how to do that ?!

**Reply by rizkyhaksono** [reply]
Thanks a lot!

**Comment by aaronlamz** [comment]
To get this badge, you need to pull requests without requiring reviews.
https://github.blog/changelog/2021-11-10-require-pull-requests-without-requiring-reviews/

**Reply by rizkyhaksono** [reply]
Thanks a lot!

**Comment by johnmichaelarc** [comment]
for Galaxy badge: Answer two discussions for Pull Shark badge :
Have two pull requests merged for yolo badge : Merge code without review
for more info : https://github.com/Schweinepriester/github-profile-achievements

**Reply by quockhanhtn** [reply]
Thank you for sharing !

**Comment by diegoleondev** [comment, marked as answer]
👋 Hi @lgrachov, These are all the achievements so far:

## Achievements

| Title | Badge | Earnable? | Earned by |
| --- | --- | --- | --- |
Pair Extraordinaire | ![Pair Extraordinaire Badge](https://github.com/Schweinepriester/github-profile-achievements/raw/main/images/pair-extraordinaire-default.png) | ✔️ | Coauthored in a merged pull request
Quickdraw | ![Quickdraw Badge](https://github.com/Schweinepriester/github-profile-achievements/raw/main/images/quickdraw-default.png) | ✔️ | Closed an issue or a pull request within 5 min of opening
Starstruck | ![Starstruck Badge](https://github.com/Schweinepriester/github-profile-achievements/raw/main/images/starstruck-default.png) | ✔️ | Created a repository that has 16 stars
||| 
Galaxy Brain | ![Galaxy Brain Badge](https://github.com/Schweinepriester/github-profile-achievements/raw/main/images/galaxy-brain-default.png) | ✔️ | 2 accepted answers
Pull Shark | ![Pull Shark Badge](https://github.com/Schweinepriester/github-profile-achievements/raw/main/images/pull-shark-default.png) | ✔️ | 2 pull requests merged
YOLO | ![YOLO Badge](https://github.com/Schweinepriester/github-profile-achievements/raw/main/images/yolo-default.png) | ✔️ | Merged a pull request without code review
||| 
Arctic Code Vault Contributor | ![Arctic Code Vault Contributor Achievement Badge](https://github.com/Schweinepriester/github-profile-achievements/raw/main/images/arctic-code-vault-contributor-default.png) | ❌ | Contributed code to repositories in the [2020 GitHub Archive Program](https://archiveprogram.github.com/)
Public Sponsor | ![GitHub Sponsor Achievement Badge](https://github.com/Schweinepriester/github-profile-achievements/raw/main/images/public-sponsor-default.png) | ✔️ | Sponsoring open source work via [GitHub Sponsors](https://github.com/sponsors)
Mars 2020 Contributor | ![Mars 2020 Helicopter Contributor Achievement Badge](https://github.com/Schweinepriester/github-profile-achievements/raw/main/images/mars-2020-contributor-default.png) | ❌ | Contributed code to repositories used in the [Mars 2020 Helicopter Mission](https://github.com/readme/nasa-ingenuity-helicopter)

You can supplement the information in the following github article: https://github.com/Schweinepriester/github-profile-achievements/blob/main/README.md

>**Note**
>Please **Mark as answer** if it was helpful.

**Reply by xyzuan** [reply]
Hiya hiya

**Reply by rizkyhaksono** [reply]
> Hiya hiya

Halo bang jody wkwk

**Comment by aninarafath6** [comment]
good

**Comment by isharax9** [comment]
> I'm new, so how do I get the badge?

Here is the answer short and sweet, 

1. Create a new pull request in your repo and open it
2. In the top right corner at Reviewers section, request someone to review your pull request
3. Merge the pull request without finishing the review.

Please mark this as answer if it helps ❤️
